RAppleWin/source/RetroAchievements.cpp

```diff
@@ -161,9 +166,16 @@ void RA_InitShared()
 
 void RA_InitSystem()
 {
-    RA_Init(g_hFrameWindow, RA_AppleWin, RAPPLEWIN_VERSION);
-    RA_InitShared();
-    RA_AttemptLogin(true);
+    if (is_initialized)
```

Comment:
There doesn't seem to be anything setting this to true. You can see the side-effects as the "Welcome User" being shown a second time, and the images not appearing.

Reply:
Oops.

Reply:
It should work as expected now.